specs/agents/tracing-spans.md

```diff
@@ -54,7 +54,36 @@ The documentation should clarify that spans with `unknown` outcomes are ignored
 
 ### Span stack traces
 
-Spans may have an associated stack trace, in order to locate the associated source code that caused the span to occur. If there are many spans being collected this can cause a significant amount of overhead in the application, due to the capture, rendering, and transmission of potentially large stack traces. It is possible to limit the recording of span stack traces to only spans that are slower than a specified duration, using the config variable `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION`.
+Spans may have an associated stack trace, in order to locate the associated
+source code that caused the span to occur. If there are many spans being
+collected this can cause a significant amount of overhead in the application,
+due to the capture, rendering, and transmission of potentially large stack
+traces. It is possible to limit the recording of span stack traces to only
+spans that are slower than a specified duration, using the config variable
+`span_stack_trace_min_duration`. (Previously
+`span_frames_min_duration`.)
+
+#### `span_stack_trace_min_duration` configuration
+
+Sets the minimum duration of a span for which stack frames/traces will be
+captured.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | [duration](configuration.md#configuration-value-types) |
+| Default        | `5ms`  |
```

Comment:
Do we want to require a common default value on all agents? Currently the node.js agent is using 10ms... which frankly was selected as "we need to be longer than the Java agent's 5ms, because stacktrace collection in Node.js takes way longer, so a 5ms default threshold doesn't help that much". However that 10ms was a scientifically selected number. My gut is that it should be even higher for node.js by default.

Reply:
I'll specify that it's a soft default, similar to log_level.